tck/src/main/java/org/eclipse/microprofile/opentracing/tck/application/TestServerWebServices.java

```diff
@@ -190,7 +320,8 @@ public Response error() {
     public Response nested(@QueryParam(PARAM_NEST_DEPTH) int nestDepth,
             @QueryParam(PARAM_NEST_BREADTH) int nestBreadth,
             @QueryParam(PARAM_UNIQUE_ID) String uniqueID,
```

Comment:
The runNestedTests test creates a uniqueID for each request (some random number) and adds it as a query parameter. Then it checks that the span results include each of these uniqueIDs. We could get rid of it by just creating a list of expected SpanTrees and just doing a compare with the list of root spans, but then it would make debugging harder because the tester would need to rummage through 100 entries to find the difference. In this way, it will print exactly which span doesn't match.